Gemfile.lock

```diff
@@ -177,31 +184,31 @@ GEM
       net-smtp
     marcel (1.0.2)
     method_source (1.0.0)
-    mini_mime (1.1.2)
-    mini_portile2 (2.8.4)
-    minitest (5.19.0)
+    mini_mime (1.1.5)
+    minitest (5.20.0)
     multi_json (1.15.0)
     multipart-post (2.3.0)
     mustermann (3.0.0)
       ruby2_keywords (~> 0.0.1)
     mustermann-grape (1.0.2)
       mustermann (>= 1.0.0)
-    net-imap (0.3.7)
+    net-imap (0.4.7)
       date
       net-protocol
     net-pop (0.1.2)
       net-protocol
-    net-protocol (0.2.1)
+    net-protocol (0.2.2)
       timeout
-    net-smtp (0.3.3)
+    net-smtp (0.4.0)
       net-protocol
     newrelic_rpm (8.12.0)
-    nio4r (2.5.9)
-    nokogiri (1.15.3)
-      mini_portile2 (~> 2.8.2)
+    nio4r (2.7.0)
+    nokogiri (1.15.5-arm64-darwin)
```

Comment:
We don't want to be committing platform-specific changes... I'm not sure why this is happening, but I can't replicate locally.  You may want to check `bundle config`, confirm you're using a ruby version manager, and/or do some additional digging to get to the bottom of this.  This is also what's causing the unwanted updates to `Platforms` below.

Reply:
@lorawoodford - This makes sense. In previous projects where we use Docker environments, this doesn't come up much. I can change this.